Server/Hambasafe.Api/appsettings.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
   "Data": {
     "DefaultConnection": {
-      "ConnectionString": "Server=localhost\\SQLEXPRESS;Database=hsdevdb1;Trusted_Connection=True;"
+      "ConnectionString": "Server=(localdb)\\MSSQLLocalDB;Database=hsdevdb1;Trusted_Connection=True;"
```

Comment:
I'm making an assumption here that everyone might not be running SQL Express on their local environment.  Visual Studio 2015 creates a localdb environment that can be used to host the dev database.